cves/CVE-2017-16840.yml

```diff
@@ -201,8 +218,10 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    There was not a mention of a violation of specification in the code (or anywhere 
+    in the repository).
```

Comment:
The commit message for the fixing commit: 
`    Ideally, there should be no temporary buffer but the encoder is designed
    to deinterleave the coefficients into the classical wavelet structure
    with the lower frequency values in the top left corner.

    Signed-off-by: Rostislav Pehlivanov <atomnuker@gmail.com>`
    This implies that they did violate specification.